### DISCUSSION TITLE
Cutting text in style with Content Margins

### DISCUSSION THREAD

**Discussion by diklor** [godotengine/godot-proposals#10448] [GUI]
### Describe the project you are working on

UI
Godot 4.3 stable

### Describe the problem or limitation you are having in your project

The last letters are cut off for different styleboxes in the button.
Disable the button, change the text, enable

### Describe the feature / enhancement and how it helps to overcome the problem or limitation

Don't cut

### Describe how your proposal will work, with code, pseudo-code, mock-ups, and/or diagrams

![issue](https://github.com/user-attachments/assets/84c57a5d-720c-478a-afda-53afdc4ede21)

Disabled (Empty)             |  Normal
:-------------------------:|:-------------------------:
![](https://github.com/user-attachments/assets/3902cd21-497d-4b75-8c17-8c7ef6b060f5)  |  ![](https://github.com/user-attachments/assets/cb25c318-1c1e-4a87-bb69-9685557c3630)

[issue_scene.zip](https://github.com/user-attachments/files/16628057/issue.zip)

### If this enhancement will not be used often, can it be worked around with a few lines of script?

No

### Is there a reason why this should be core and not an add-on in the asset library?

Core

**Comment by AThousandShips** [comment]
Converted to a discussion due to lack of details in proposal

**Comment by diklor** [comment]
Empty project, just create a scene with a container, in container button with both central alignment. When the button is disabled and the text is changed when it's disabled, when set 'Disabled ' to false, the last characters are cut

I noticed immediately after the test of my project from 4.2.2 to 4.3 stable, I did nothing with the scenes

**Reply by Calinou** [reply]
This sounds like a bug. Bugs should be reported on the [main Godot repository](https://github.com/godotengine/godot), not on godot-proposals.

- **Edit:** Issue opened: https://github.com/godotengine/godot/issues/95580

**Comment by diklor** [comment]
At runtime text is cut off simply by setting 'Disabled' off from script (`button.disabled = false`) when the button was initially disabled in the scene
